src/entity-manager/EntityManager.ts

```diff
@@ -945,6 +945,23 @@ export class EntityManager {
         }
     }
 
+    /**
+     * Checks whether any entity exists with the given condition
+     */
+    exists<Entity>(
```

Comment:
please also add tests checking this function and testing different `options?: FindManyOptions<Entity>`